numbert/utils_reranker.py

```diff
@@ -87,15 +95,24 @@ def online_eval(model, tokenizer, device, features, batch_size = 1, model_type="
     numbert_predictions = {}
     # numbert_labels = {}
     for ind, guid in enumerate(lguids):
-        # if out_label_ids[ind] == 1:
-        #     if guid[1] in numbert_labels:
-        #         numbert_labels[guid[1]].add(guid[3])
-        #     else:
-        #         numbert_labels[guid[1]] = {guid[3]}
-        if guid[1] in numbert_predictions:
-            numbert_predictions[guid[1]].append((guid[3], preds[ind][1]))
+        if is_duoBERT:
+            if guid[1] in numbert_predictions:
```

Comment:
This block is pretty hard to parse. Maybe rewrite it or add comments explaining what it does?